discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -129,22 +129,18 @@ def select_installation_disk(self, hosts_with_disk_paths):
         self.api_client.select_installation_disk(self.id, hosts_with_disk_paths)
 
     def set_ocs(self, ocs_enabled):
-        logging.info(f'Enabling Ocs to:{ocs_enabled} for cluster: {self.id}')
         self.set_operator('ocs', ocs_enabled)
 
-    def set_cnv(self, cnv_enabled):
-        logging.info(f'Enabling CNV to:{cnv_enabled} for cluster: {self.id}')
-        self.set_operator('cnv', cnv_enabled)
-
     def set_operator(self, operator, enabled):
+        logging.info(f'Enabling {operator} to:{enabled} for cluster: {self.id}')
         cluster = self.api_client.cluster_get(self.id)
-        olm_operators = cluster.olm_operators
-        olm_names = [olm_operator['name'] for olm_operator in olm_operators]
-        if operator not in olm_names and enabled:
-            olm_operators.append({'name': operator})
-        if operator in olm_names and not enabled:
-            olm_operators.remove({'name': operator})
-        self.api_client.update_cluster(self.id, olm_operators)
+        monitored_operators = cluster.monitored_operators
+        operator_names = [monitored_operator['name'] for monitored_operator in monitored_operators]
+        if operator not in operator_names and enabled:
+            monitored_operators.append({'name': operator})
+        if operator in operator_names and not enabled:
+            monitored_operators.remove({'name': operator})
+        self.api_client.update_cluster(self.id, monitored_operators)
```

Comment:
I don't we can directly update `cluster.monitored_operators` , we need to update the cluster as: ```self.api_client.update_cluster(self.id, {"olm_operators":[{"name":"ocs"}]})```
which in general form can be:
```self.api_client.update_cluster(self.id, {"olm_operators":<updated_array>})``` .
we could extract the list of operators in the `monitored_operators` , and amend in the new array, then pass that to update the cluster

Reply:
Oh right. Also I just realized, I need to pass also `properties` parameter, or it will be removed. Thanks for review so far.

Reply:
for enabling, the properties field doesn't matter. Through api, I was able to enable the operators with just passing the name field

Reply:
Yep, but imagine scenario like this:
```
cluster.set_operator('ocs', properties='something')
cluster.set_operator('cnv')
```

With this flow we set OCS OLM with some properties, and then when we enabled the CNV as well, we remove the OCS properties. Which is wrong. Unless we don't care about `properties` field in the tests for now.

Reply:
Yes, the operator that is already in `montitored_operators` should be preserved, while for enabling new operators, only name can be included